content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -579,6 +579,15 @@ query.iter().for_each(|mut component| {
 
 [`Query::par_iter`]: https://docs.rs/bevy/0.10.0/bevy/ecs/system/struct.Query.html#method.par_iter
 
+
+## `UnsafeWorldCell` and `UnsafeEntityCell`
+
+<div class="release-feature-authors">authors: @jakobhellermann, @BoxyUwU and @JoJoJet</div>
+
+`UnsafeWorldCell` and `UnsafeEntityCell` allow shared mutable access to parts of the world via unsafe code. It serves a similar purpose as `UnsafeCell`, allowing people to build interior mutability abstractions such as `Cell` `Mutex` `Channel` etc. In bevy `UnsafeWorldCell` will be used to support the scheduler and system param implementations as these are interior mutability abstractions for `World`, it also currently is used to implement `WorldCell`. It's also likely that `UnsafeEntityCell` will be used to implement versions of `EntityRef`/`EntityMut` that only have access to the components on the entity rather than the entire world.
```

Comment:
```suggestion
`UnsafeWorldCell` and `UnsafeEntityCell` allow shared mutable access to parts of the world via unsafe code. It serves a similar purpose as `UnsafeCell`, allowing people to build interior mutability abstractions such as `Cell` `Mutex` `Channel` etc. In bevy `UnsafeWorldCell` will be used to support the scheduler and system param implementations as these are interior mutability abstractions for `World`, it also currently is used to implement `WorldCell`. We're planning to use `UnsafeEntityCell` to implement versions of `EntityRef`/`EntityMut` that only have access to the components on the entity rather than the entire world.
```